src/AppKit/Enums.cs

```diff
@@ -2112,6 +2122,7 @@ public static class NSFileTypeForHFSTypeCode {
 	
 	// These constants specify the possible states of a drawer.
 	[Native]
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Drawers are deprecated; consider using 'NSSplitViewController'")]
```

Comment:
I copy pasted all of the spam from the various deprecation attributes and then went back and fixed things like 's and such, but missed ones like this who's text is completely wrong.